stdlib/contrib/bonitoo-io/alerta/alerta.flux

```diff
@@ -1,25 +1,71 @@
+// Package alerta provides functions that send alerts to [Alerta](https://alerta.io/).
+//
+// introduced: 0.115.0
 package alerta
 
 
 import "http"
 import "json"
 import "strings"
 
-// alert sends an alert to Alerta.
-// `url` - string - Alerta URL.
-// `apiKey` - string - Alerta API key.
-// `resource` - string - resource under alarm.
-// `event` - string - event name.
-// `environment` - string - environment. Valid values: "Production", "Development" or empty string (default).
-// `severity` - string - event severity. See https://docs.alerta.io/en/latest/api/alert.html#alert-severities.
-// `service` - arrays of string - list of affected services.
-// `group` - string - event group.
-// `value` - string - event value.
-// `text` - string - text description.
-// `type` - string - event type.
-// `origin` - string - monitoring component.
-// `timestamp` - time - time alert was generated.
-// `timeout` - int - seconds before alert is considered stale.
+// alert sends an alert to [Alerta](https://alerta.io/).
+//
+// ## Parameters
+//
+// - url: - string - (Required) Alerta URL.
+// - apiKey: - string - (Required) Alerta API key.
+// - resource: - string - (Required) Resource associated with the alert.
+// - event: - string - (Required) Event name.
+// - environment: - string - Alerta environment. Valid values: "Production", "Development" or empty string (default).
+// - severity: - string - (Required) Event severity. See [Alerta severities](https://docs.alerta.io/en/latest/api/alert.html#alert-severities).
+// - service: - arrays of string - List of affected services. Default is `[]`.
+// - group: - string - Alerta event group. Default is `""`.
+// - value: - string - Event value.  Default is `""`.
+// - text: - string - Alerta text description. Default is `""`.
+// - tags: - array of strings - List of event tags. Default is `[]`.
+// - attributes: - record - (Required) Alert attributes.
+// - origin: - string - monitoring component.
+// - type: - string - Event type. Default is `""`.
+// - timestamp: - time - time alert was generated. Default is `now()`.
+// 
+// ## Examples
+// ### Send the last reported value and status to Alerta
+// ```
```

Comment:
can you add a `no_run` to this example as we don't want it to try to connect to alerta?